packs/data/pfs-season-1-bestiary.db/guard-pfs-1-17.json

```diff
@@ -1091,7 +1091,7 @@
             "type": "lore"
         }
     ],
-    "name": "Guard (1-17)",
+    "name": "Guard (PFS 1-17)",
```

Comment:
Difference from Bestiary version: 
Fey trait
Mwangi and Sylvan languages
AC
HP
Fort and Ref saves
No dagger or signal whistle
Scale mail switched to chain
Simple manacles